### ISSUE TITLE
VaButton redesign

### ISSUE THREAD

**Issue by asvae** [epicmaxco/vuestic-ui#1906]
We have Vasily design on new iteration of buttons: [figma](https://www.figma.com/file/2yH0KFiWds3q1B5NA5dELZ/vuestic-ui?node-id=1538%3A7170).

- [ ] Sort requirements in [notion](https://www.notion.so/epicmax/Vuestic-ui-10e698cacd5846688647e1d8482277c9?p=43340df8b3f8476ab33672ade22d50e3).
- [ ] Create new issue with imlementation requirements.
- [ ] Assign someone to review this issue before starting with implementation.

**Comment by m0ksem** [comment]
Make sure to change `:rounded="false"` to `square`. Same as VaChip.

**Comment by aluarius** [comment]
Implementation issue: https://github.com/epicmaxco/vuestic-ui/issues/1923
